lib/ansible/executor/action_write_locks.py

```diff
@@ -36,8 +37,7 @@
 
     # These plugins are called directly by action plugins (not going through
     # a strategy).  We precreate them here as an optimization
-    mods = set(p['name'] for p in Facts.PKG_MGRS)
+    mods = set(C.PKG_MGR_MODULES)
```

Comment:
I don't think the data belongs in constants.py.  Using Facts.PKG_MGRS was an attempt to make a single canonical source for the data.  We should probably derive this from ```ansible.module_utils.facts.system.pkg_mgr.PKG_MGRS``` now.

Tangent: Looking at this now, I realize that when I coded this, I was overly optimistic about the data being usable in this way.  Not everything in C.PKG_MGR_MODULES is actually a module.  OTOH, it's not a big problem.  We just add a few more dictionary keys that map to other Lock() objects.  (this is a, preexisting problem, doesn't need to be solved)